app/views/layouts/_review_form.html.erb

```diff
@@ -1,3 +1,8 @@
+<% if controller.action_name == "show" %>
+<h2>Write a new review</h2>
+<%elsif controller.action_name == "edit" %>
+<h2>Edit your review</h2>
+<%end%>
 <%= form_for [@museum, @review] do |form| %>
```

Comment:
I wouldn't keep partials in the layouts folder. This should either be in the review views folder or in a new views folder called 'shared'.